NuGet.Config

```diff
@@ -0,0 +1,20 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+	<packageSources>
+		<add key="NugetOfficialV3" value="https://api.nuget.org/v3/index.json" />
+		<add key="Local" value="tools/LocalNugetFeed" />
+	</packageSources>
+	
+	<config>
+		<add key="globalPackagesFolder" value="restoredPackages" />
+		<add key="RestorePackagesPath" value="restoredPackages" />
+	</config>
+  
+	<packageRestore>
+		<!-- Allow NuGet to download missing packages -->
```

Comment:
I suspect these settings aren't needed.

Reply:
I can remove it, I just thought those are global properties that our repo can benefit, but if it's not requried, I will remove it